modules/rosa-sts-account-roles-terraform.adoc

```diff
@@ -0,0 +1,193 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.20.0"
+    }
+    rhcs = {
+      version = ">= 1.3.0"
+      source  = "terraform-redhat/rhcs"
+    }
+  }
+}
+
+data "rhcs_policies" "all_policies" {}
+
+data "rhcs_versions" "all" {}
+
+module "create_account_roles" {
+  source  = "terraform-redhat/rosa-sts/aws"
+  version = "0.0.15"
+
+  create_operator_roles = false
+  create_oidc_provider  = false
+  create_account_roles  = true
+
+  account_role_prefix    = var.account_role_prefix
+  rosa_openshift_version = var.openshift_version
+  account_role_policies  = data.rhcs_policies.all_policies.account_role_policies
+  operator_role_policies = data.rhcs_policies.all_policies.operator_role_policies
+  all_versions           = data.rhcs_versions.all
+  tags                   = var.tags
+}
+----
+
+.. Copy this template into a file  named `output.tf`. In the `output.tf` file, the account role prefix structure is defined. This output definition allows you to specify how the various generated roles are constructed. 
++
+[source,terraform]
+----
+output "account_role_prefix" {
+  value = module.create_account_roles.account_role_prefix
+}
+----
+
+.. Copy this template into a file  named `variables.tf`. You can provide your own variables in the default fields for the account-role creation variables. 
++
+[IMPORTANT]
+====
+Do not include your OCM token in this file if it is not stored in a safe location.
+====
++
+[source,terraform]
+----
+variable "openshift_version" {
+  type = string
+  default = "4.13"
+  description = "Enter the desired OpenShift version as X.Y. This version should match what you intend for your ROSA cluster. For example, if you plan to create a ROSA cluster using `4.13.10`, then this version should be `4.13`. You can see the supported versions of OpenShift by running `rosa list version`."
+}
+
+variable "account_role_prefix" {
+  type    = string
+  default = ""
+  description = "Your account roles are prepended with whatever value you enter here. The default value in the ROSA CLI is `ManagedOpenshift-` before all of your account roles."
+}
+
+variable "tags" { <1>
+  description = "(Optional) List of AWS resource tags to apply."
+  type        = map
+  default     = null
+}
+----
++
+--
+<1> The `tags` parameter uses a map of strings variable type. The format that it takes looks like the following example:
++
+[source,terminal]
```

Comment:
Still not sure if [source,terraform] is a valid option but if it is, shouldn't this block also be the same?